Meta/gn/secondary/Userland/Libraries/LibTimeZone/BUILD.gn

```diff
@@ -0,0 +1,25 @@
+import("//Meta/gn/build/library_name.gni")
+
+declare_args() {
+    enable_timezone_database_download = false
```

Comment:
If you add a one-line comment above this, it'll be printed as help on `gn args --list out/gn`